scripts/container_start.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/sh
-export DEPLOYMENT=${DEPLOYMENT:-api}
+DEPLOYMENT=${1:-api}
```

Comment:
Why removed export?

Reply:
why we want it to be global?

Reply:
So that we can know inside the container which one it is